packages/cli/src/cmds/validator/options.ts

```diff
@@ -277,6 +288,14 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
     group: "externalSignerUrl",
   },
 
+  // Distributed validator
+
+  distributed: {
+    description: "Enables specific features required to run as part of a distributed validator cluster",
+    default: false,
```

Comment:
No need to set the default value to the primitive's default value, please remove

Reply:
Reasoning for setting this was to improve clarity on what the default is and don't we show this in the cli docs?

Reply:
ok, seems to be more consistent with other `boolean` values, I removed it